deployment/roles/user-traffic-reader/tasks/main.yml

```diff
@@ -16,7 +17,7 @@
     user: root
     tty: True
     detach: no
-    image: quay.io/3scale/perftest-toolkit:buddhi-v2-latest
+    image: quay.io/3scale/perftest-toolkit:master
```

Comment:
Same as other comments related to perftest-toolkit version. In case some changes related to this this should be updated too

Reply:
same answer :)